src/components/proxy-middleware/helpers/harObectCreator.js

```diff
@@ -12,8 +12,28 @@ const createHarHeaders = (request_headers_obj) => {
   return headers;
 };
 
-const createHarQueryStrings = (path) => {
+const createHarQueryStrings = (path, query_params) => {
   // TODO -> http://www.softwareishard.com/blog/har-12-spec/#queryString
+  console.log("path", path)
+  console.log("query_params", query_params)
+  if(path) {
+    const start = path.indexOf("?")
+    if(start !== -1) {
+      const subPath = path.substring(start+1)
+      console.log("subPath",subPath)
+
+      const result = subPath.split("&").map(queryPairs => {
+        const queryParts = queryPairs.split("=")
+        if(queryParts.length == 2) {
+          return {
+            "name": queryParts[0],
+            "value": queryParts[1]
+          }
+        }
+      })
+      return result
+    }
+  }
```

Comment:
Why are we creating queryParams here using path. Can't we use the `query_params` map created earlier?